Add subsections in 'For users'

pireddag opened the pull request.

pireddag committed:
Added subsections in 'For Users' section

pireddag committed:
Added subsections in "For user"

pireddag committed:
Revert "Added subsections in 'For Users' section"

This reverts commit f54d8952793a923c431d80ab00987e359c2478a5.

pireddag commented:
Second test.

mgubi commented on src/awesome-texmacs.tm:
```diff
@@ -1,11 +1,13 @@
 <TeXmacs|1.99.14>
 
-<style|notes>
+<style|browser>
```
Why you change this package? This has to be left at `notes`.

pireddag replied:
The change happened without me realizing it, I could start everything from scratch as I do not know why the change is there

mgubi replied:
Maybe you can try to change it on a text editor and open again the file to see if TeXMacs changes it again. Apriori there should be no need for that. You see "Notes" as the style of the document if all is correct.

mgubi replied:
Just close this PR and open another one where you have one commit. We can use these try to see and write down some precise instructions on how to contribute.

pireddag replied:
Ok, I'll do that.

mgubi commented on src/awesome-texmacs.tm:
```diff
@@ -1,11 +1,13 @@
 
 <\body>
-  <hlink|[index]|./index.tm><chapter*|Awesome GNU TeXmacs>
+  <hlink|[index]|./index.html><label|auto-1>
```
Also here. Please leave the link to .tm, the conversion is done automatically later on.

mgubi commented on src/awesome-texmacs.tm:
```diff
@@ -1,11 +1,13 @@
-  <section*|Plugins>
+  <chapter*|<label|auto-1>Awesome GNU TeXmacs>
+
+  <section*|<label|auto-2><label|auto-2>Plugins>
```
If you do not make meaningful changes, then better to remove it from your commit. Like here?

pireddag replied:
All of these changes happened without me realizing it. Perhaps it is better if I re-fork the repository and start from scratch, but I need to understand what happened, otherwise the program will keep behaving in the same way.

pireddag committed:
More detail on git

Link to a tutorial, brief instructions on pull requests. I changed "Clone the repository" with "Fork the repository", as I found that I could send a pull request only from a fork (learnt today the git commands), but maybe I am wrong there.

mgubi commented:
Dear Giovanni, a general remark: I think PR are meant to be done as follows: 1) you clone the repository 2) You make your changes in one go (no multiple commit, unless there is some specific reason) 3) you ask the PR. In this way the history is cleaner.

pireddag commented:
In summary, what I tried to do
* do two separate pull requests, one for awesome-texmacs, one for the readme file

what I actually did was a bit different :-)

Also I did not yet understand whether I should *clone* or *fork* the repository. As far as I see I clone *my* repository and I push changes to it, but I have to fork *someone else's* repository, then clone the forked repository, push changes to that and finally make a pull request on the original repository. But I haven't yet found an authoritative source for that.

mgubi commented:
> In summary, what I tried to do
> 
> * do two separate pull requests, one for awesome-texmacs, one for the readme file
> 
> what I actually did was a bit different :-)
> 
> Also I did not yet understand whether I should _clone_ or _fork_ the repository. As far as I see I clone _my_ repository and I push changes to it, but I have to fork _someone else's_ repository, then clone the forked repository, push changes to that and finally make a pull request on the original repository. But I haven't yet found an authoritative source for that.

Look here for example: https://www.freecodecamp.org/news/how-to-make-your-first-pull-request-on-github-3/

you have to clone the repository you want to make a PR to, then create a new branch in your clone and commit to this branch your changes, then push the branch to upstream. This will create a PR to merge your branch. (I'm a newby too, so take all this with a grain of salt)

pireddag commented:
The link https://www.freecodecamp.org/news/how-to-make-your-first-pull-request-on-github-3/ starts with 

1. Fork the repository

pireddag reviewed (commented):
I want to approve the changes but the option "Approve" is not active. Let me figure that out and approve the changes, then I'll figure out why the extra modifications happened.

pireddag commented:
Close pull request, restart from scratch.
